frameworks/cassandra/tests/test_recovery_shutdown.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from tests.config import *
+import sdk_install as install
+import sdk_tasks as tasks
+import sdk_utils as utils
+import json
+import shakedown
+import time
+import sdk_cmd as cmd
+
+
+def setup_module(module):
+    install.uninstall(PACKAGE_NAME)
+    utils.gc_frameworks()
+
+    # check_suppression=False due to https://jira.mesosphere.com/browse/CASSANDRA-568
+    install.install(PACKAGE_NAME, DEFAULT_TASK_COUNT, check_suppression=False)
+
+
+def setup_function(function):
+    tasks.check_running(PACKAGE_NAME, DEFAULT_TASK_COUNT)
+
+
+def teardown_module(module):
+    install.uninstall(PACKAGE_NAME)
+
+
+@pytest.mark.sanity
+@pytest.mark.recovery
+def test_terminate_host_test():
+
+    service_ip = shakedown.get_service_ips(PACKAGE_NAME).pop()
+    print('marathon ip = {}'.format(service_ip))
+
+    node_ip = 0
+    for pod_id in range(0, DEFAULT_TASK_COUNT):
+        node_ip = get_pod_host(pod_id)
+        if node_ip != service_ip:
+            break
+
+    if node_ip is None:
+        return
```

Comment:
Will this fail the test? Because the test _should_ fail if it can't determine an agent to terminate right?